cross-platform/startuptimes/startuptimes.py

```diff
@@ -0,0 +1,582 @@
+#!/usr/bin/env python3
+
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+# See LICENSE.md in the project root for license terms and full copyright notice.
+#---------------------------------------------------------------------------------------------
+
+'''
+Application to maintain a database of startup times for the iTwin Mobile SDK
+samples.
+'''
+
+import argparse
+import sqlite3
+import json
+import sys
+import textwrap
+from typing import Any, Callable, Sequence, TextIO, Union
+from io import TextIOWrapper
+from functools import partial
+
+Record = dict[str, Any]
+Records = Sequence[Record]
+Test = Union[tuple[str, str], str]
+ColumnDef = Union[str, tuple[str, str]]
+ColumnDefs = Sequence[ColumnDef]
+Formatters = Sequence[Callable[[Any], str] | None]
+SQLiteDescription = tuple[tuple[str, None, None, None, None, None, None], ...] | Any
+
+class ASCIITable:
```

Comment:
This name seems a bit confusing to me. It's not limited to ASCII characters, is it? Perhaps `ReportTable` would be better?

Reply:
I renamed the class to TextTable. Let me know if you don't like that name.